src/resources/common/Configuration.ts

```diff
@@ -169,6 +169,23 @@ export interface InstallmentsConfiguration {
 
 export interface InstallmentsConfigurationItem extends PaymentTypeConfiguration, InstallmentsConfiguration {}
 
+export type SubscriptionsPlanConfiguration = {
```

Comment:
very minor but if a variable name consists of multiple words I m more used to having them in their singular form and then use plural (if needed) on the last one. So here I would go with `SubscriptionPlanConfiguration`.
But seems like we have the same naming pattern for other `interface` names in this file as well so if we want to keep it consistent maybe we don't need to change 🙇

Reply:
Ups, thanks! That is correct, the type should follow the property name. Thanks for catching it.

Reply:
thanx! resolving!